tests/components/pi_hole/test_init.py

```diff
@@ -182,12 +184,13 @@ async def test_unload(hass: HomeAssistant) -> None:
     with _patch_init_hole(mocked_hole):
         await hass.config_entries.async_setup(entry.entry_id)
         await hass.async_block_till_done()
-    assert entry.entry_id in hass.data[pi_hole.DOMAIN]
+    assert entry.state == ConfigEntryState.LOADED
```

Comment:
```suggestion
    assert entry.state is ConfigEntryState.LOADED
```
Use `is` when comparing with an enum value.